app/src/components/market/market_home_container.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+import { MarketHome } from './market_home'
+import { useConnectedWeb3Context } from '../../hooks/connectedWeb3'
+import { useMarkets } from '../../hooks/useMarkets'
+
+interface Props {
+  marketMakerAddress: string
+}
+
+const MarketHomeContainer = () => {
+  const context = useConnectedWeb3Context()
+
+  const { markets, status } = useMarkets(context)
```

Comment:
Is this hook going to be used elsewhere? If not, what's the point of having a hook for this?

Reply:
Now it is used in two components, so I think it is necessary